src/postgres/commands/PostgresAccountWizard/PostgresServerCredPWStep.ts

```diff
@@ -15,7 +15,10 @@ export class PostgresServerCredPWStep extends AzureWizardPromptStep<IPostgresWiz
         const user = nonNullProp(wizardContext, 'adminUser');
         wizardContext.adminPassword = (await ext.ui.showInputBox({
             placeHolder: localize('pwPlaceholder', 'Password'),
-            prompt: localize('enterPWPrompt', 'Enter administrator password for the server.'),
+            prompt: localize('enterPWPrompt', 'Enter administrator password for the server. ' +
```

Comment:
Don't add strings together like this inside `localize`. I know it makes it look a little cleaner, but I don't think the localization tools work unless it's one constant string